test/apps/react-routing/react-router-dom-v5/src/components/SecureRoute.tsx

```diff
@@ -0,0 +1,76 @@
+/*!
+ * Copyright (c) 2017-Present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import React from 'react';
+import { useOktaAuth } from '@okta/okta-react';
+import { toRelativeUrl } from '@okta/okta-auth-js';
+import { Route, RouteProps, useRouteMatch } from 'react-router-dom';
+
+export const SecureRoute: React.FC<{
```

Comment:
would it be feasible to ship both `SecureRoute` and `SecureRoutes` under respective router subdirectories as a part of SDK? (we could probably mark `reach-router` and `react-router-dom` as [optional](https://yarnpkg.com/configuration/manifest#peerDependenciesMeta) peer dependencies)

Reply:
Our plan is to remove `SecureRoute` from the sdk altogether and then use these test apps as samples/examples of how one would implement routing with the various react routing libraries

Reply:
any movement here?

Reply:
@tehhowch yes, https://github.com/okta/okta-react/tree/master/samples/routing